feature-libs/product/future-stock/components/future-stock/future-stock.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-container *ngIf="futureStocks">
+  <cx-future-stock-accordion
+    [header]="'futureStockDropdown.header' | cxTranslate"
+    [content]="futureStocks"
```

Comment:
Can we pass the future stocks observable instead and have the logic handled inside the accordian component?

Reply:
I would rather keep the container component smart, and accordion dumb, here is why:
https://blog.angular-university.io/angular-2-smart-components-vs-presentation-components-whats-the-difference-when-to-use-each-and-why/